### PR TITLE
feat: flipt feature added for feature enable and disable

### PR DESCRIPTION
**Pull request by NiravThumar**
ISSUE LINK :  https://github.com/Improwised/golang-api/issues/226

PR for integrate Flipt to golang boilerplate

### CODE REVIEW

File: cli/api.go [hunk 1]
```diff
@@ -44,12 +45,23 @@ func GetAPICommandDef(cfg config.AppConfig, logger *zap.Logger) cobra.Command {
...
-			pub, err := watermill.InitPublisher(cfg,false)
+			pub, err := watermill.InitPublisher(cfg, false)
 			if err != nil {
 				return err
 			}
+
+			// Initialize Flipt client for flipt functionality
+			err = helpers.InitFliptClient()
+			if err != nil {
+				logger.Error("Error while initlize client", zap.Error(err))
```
Comment (munir131) [inline cli/api.go:56]: @NiravThumar Typo in initialize

File: docker-compose.yaml [hunk 2]
```diff
@@ -55,5 +55,32 @@ services:
...
 
+  init:
+    image: flipt/flipt:latest
+    profiles: [flipt]
+    command: ["./flipt", "import", "flipt.yml"]
+    environment:
+      - FLIPT_LOG_LEVEL=debug
+      - FLIPT_META_TELEMETRY_ENABLED=false
+    volumes:
+      - "./flipt.yml:/flipt.yml"
```
Comment (munir131) [inline docker-compose.yaml:66]: @NiravThumar Please add sample flipt.yml too for easy setup

**Review by munir131 — CHANGES_REQUESTED** [review]

File: cli/api.go (hunk 1 above)
Comment (NiravThumar) [reply cli/api.go]: done

File: docker-compose.yaml (hunk 2 above)
Comment (NiravThumar) [reply docker-compose.yaml]: done

**Review by munir131 — APPROVED** [review]

**Pull request merged**